src/App.jsx

```diff
@@ -0,0 +1,16 @@
+import { useState } from 'react';
+import Page from './Page';
+
+export default function App() {
+  const [state, setState] = useState({ count: 0 });
+  const { count } = state;
+  function onClickOne() {
+    setState((prev) => ({ ...prev, count: count + 1 }));
+  }
+  function onClickNumbers(numbers) {
+    setState((prev) => ({ ...prev, count: count + numbers }));
+  }
+  return (
+    <Page count={count} onClickOne={onClickOne} onClickNumbers={onClickNumbers} />
```

Comment:
```suggestion
    <Page
      count={count}
      onClickOne={onClickOne}
      onClickNumbers={onClickNumbers}
    />
```

저는 속성이 2개 이상일 때는 줄바꿈해주면 코드가독성이 더 좋다고 생각하고 코드를 정리해주는 편이에요.
코드 가독성을 고려한 자기만의 코드 정리 기준을 세우시면 좋아요.